docs/products/calloption.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Call Options
+sidebar_label: Call Options
+---
+A call option is a building block for the derivatives market. It gives the buyer the right but not the obligation to buy the asset at a specific price (strike) at a specific time. 
```

Comment:
Perhaps we want to put something about why it is easy to build call options on UMA within the introduction.